homework/style.css

```diff
@@ -1,3 +1,121 @@
+* {
+  margin: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+  margin-bottom: 100px;
+  color: rgb(36, 63, 111);
+  background: rgb(216, 247, 254);
+  background: radial-gradient(
+    circle,
+    rgba(216, 247, 254, 0.9976365546218487) 29%,
+    rgba(80, 117, 244, 1) 100%
+  );
+}
+
+#root {
+  width: 80%;
+  margin: 0 auto;
+}
+
 .alert-error {
-    color: red;
-}
\ No newline at end of file
+  color: red;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+a {
+  padding: 0;
+  color: rgb(25, 92, 150);
+}
+
+a:hover {
+  font-size: 1.1em;
+}
+
+img {
+  width: 70px;
+  height: 70px;
+}
+
+header {
+  display: block;
+  padding: 3em;
+  align-content: center;
+  text-align: center;
+  text-shadow: 1px 3px 3px rgb(87, 68, 154);
+}
+
+h1 {
+  font-size: 3em;
+  letter-spacing: 2px;
+  color: rgb(210, 249, 222);
+}
+
+select {
+  color: rgb(36, 63, 111);
+  margin-top: 20px;
+  padding: 10px;
```

Comment:
This padding causes the text inside the select box to become unreadable. Best to remove it altogether.

Reply:
Okay thank you so much for all the feedback!